mxtest/pcas.override.yaml

```
@@ -0,0 +1,11 @@
+version: '3.8'
+
```

```suggestion
version: '3.8'

```

Docker compose considers version numbers a deprecated property.
https://github.com/docker/compose/issues/11628#issuecomment-2000072567